code/modules/mob/living/carbon/xenomorph/castes/hivemind/abilities_hivemind.dm

```diff
@@ -5,3 +5,80 @@
 
 /datum/action/xeno_action/return_to_core/action_activate()
 	SEND_SIGNAL(owner, COMSIG_XENOMORPH_CORE_RETURN)
+
+// ***************************************
+// *********** Hivemind healing
+// ***************************************
+/datum/action/xeno_action/activable/hivemind_heal
+	name = "Hivemind Heal"
+	action_icon_state = "heal_xeno"
+	mechanics_text = "Heal."
+	cooldown_timer = 1 MINUTES
```

Comment:
wait this wasn't the original cool down?

Reply:
Yes. It is the application time which is shorter. I could have the cooldown be quicker, but that might be better separated inside another PR.